cmp/grid/renderers/MultiFieldRenderer.ts

```diff
@@ -107,13 +119,19 @@ function renderSubField({colId, label}, context) {
 
     if (label && !isString(label)) label = headerName;
 
-    return div({
-        className: 'xh-multifield-renderer-field',
-        items: [label ? `${label}: ` : null, renderValue(value, renderer, column, context)]
-    });
+    const renderedVal = renderValue(value, renderer, column, context),
+        renderedValIsEmpty = renderedVal === '' || isNil(renderedVal);
+
+    return renderedValIsEmpty
+        ? null
+        : div({
+              className: 'xh-multifield-renderer-field',
+              items: [label && !renderedValIsEmpty ? `${label}: ` : null, renderedVal]
```

Comment:
second check of the boolean is now obsolete

Reply:
kk good catch thanks